### PR TITLE
Architecture cleanup + reformatting and unused import removal

### PR DESCRIPTION
**Pull request by Nodrex**

### CODE REVIEW

File: testomania/src/main/java/com/earth/testomania/common/Util.kt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import android.util.Log
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+
+const val GENERIC_TAG = "TESTOMANIA_TAG"
+
+fun log(text: String) {
+    Log.d(GENERIC_TAG, text)
+}
+
+object Crashlytics {
```
Comment (shalva97) [inline testomania/src/main/java/com/earth/testomania/common/Util.kt:12]: why not provide this via Hilt?

File: testomania/src/main/java/com/earth/testomania/di/AppModule.kt [hunk 2]
```diff
@@ -25,7 +25,7 @@ object AppModule {
 
     @Provides
     @Singleton
-    fun jsonSerializer(): Moshi {
+    fun provideJsonSerializer(): Moshi {
```
Comment (shalva97) [inline testomania/src/main/java/com/earth/testomania/di/AppModule.kt:28]: hmmm

**Review by shalva97 — APPROVED** [review]

File: testomania/src/main/java/com/earth/testomania/common/Util.kt (hunk 1 above)
Comment (Nodrex) [reply testomania/src/main/java/com/earth/testomania/common/Util.kt:12]: Yes, I also thought this, but we need it not only in view model and i do not want to inject it in every place, cause writing this is so simple. So i think for now we do not need to inject it

File: testomania/src/main/java/com/earth/testomania/di/AppModule.kt (hunk 2 above)
Comment (Nodrex) [reply testomania/src/main/java/com/earth/testomania/di/AppModule.kt:28]: Let's be consistent 😀

**Pull request merged**